keylkgSup/keylkg.db

```diff
@@ -0,0 +1,454 @@
+{
+    field(DESC, "Set offset for signal: output 1")
+    field(DTYP, "stream")
+    field(PREC, "4")
+    field(EGU, "")
+    field(DRVH, "99.9999")
+    field(DRVL, "-99.9999")
+    
+    field(OUT, "@keylkg.proto setOffsetOutput1 $(PORT)")
+    
+    field(SDIS, "$(P)DISABLE_IF_NORM_MODE")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:OFFSET:OUTPUT:1:SP")
+    info(INTEREST, "HIGH")
+}
+
+alias("$(P)OFFSET:OUTPUT:1", "$(P)OFFSET:OUTPUT:1:SP:RBV")
+
+record(ai, "$(P)OFFSET:OUTPUT:2")
+{
+    field(DESC, "Current Offset for signal: output 2")
+    field(DTYP, "stream")
+    field(SCAN, "1 second")
+    field(PREC, "4")
+    field(EGU, "")
+    
+    field(INP, "@keylkg.proto getOffsetOutput2 $(PORT)")
+    
+    field(SDIS, "$(P)DISABLE_IF_NORM_MODE")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:OFFSET:OUTPUT:2")
+    info(INTEREST, "HIGH")
+}
+
+record(ao, "$(P)OFFSET:OUTPUT:2:SP")
+{
+    field(DESC, "Set offset for signal: output 2")
+    field(DTYP, "stream")
+    field(PREC, "4")
+    field(EGU, "")
+    field(DRVH, "99.9999")
+    field(DRVL, "-99.9999")
+    
+    field(OUT, "@keylkg.proto setOffsetOutput2 $(PORT)")
+    
+    field(SDIS, "$(P)DISABLE_IF_NORM_MODE")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:OFFSET:OUTPUT:2:SP")
+    info(INTEREST, "HIGH")
+}
+
+alias("$(P)OFFSET:OUTPUT:2", "$(P)OFFSET:OUTPUT:2:SP:RBV")
+
+record(ao, "$(P)OFFSET:OUTPUT:BOTH:SP")
+{
+    field(DESC, "Set offset for signal: both outputs")
+    field(DTYP, "stream")
+    field(PREC, "4")
+    field(EGU, "")
+    
+    field(OUT, "@keylkg.proto setOffsetOutputBoth $(PORT)")
+    
+    field(SDIS, "$(P)DISABLE_IF_NORM_MODE")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:OFFSET:OUTPUT:BOTH:SP")
+    info(INTEREST, "MEDIUM")
+}
+
+alias("$(P)OFFSET:OUTPUT:BOTH:SP", "$(P)OFFSET:OUTPUT:BOTH:SP:RBV")
+
+record(ai, "$(P)VALUE:OUTPUT:1")
+{
+    field(DESC, "Measurement value for output 1")
+    field(DTYP, "stream")
+    field(SCAN, ".5 second")
+    field(PREC, "4")
+    field(EGU, "")
+    
+    field(INP, "@keylkg.proto getValueOutput1 $(PORT)")
+    
+    field(SDIS, "$(P)DISABLE_IF_COMM_MODE")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:VALUE:OUTPUT:1")
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)VALUE:OUTPUT:2")
+{
+    field(DESC, "Measurement value for output 2")
+    field(DTYP, "stream")
+    field(SCAN, ".5 second")
+    field(PREC, "4")
+    field(EGU, "")
+    
+    field(INP, "@keylkg.proto getValueOutput2 $(PORT)")
+    
+    field(SDIS, "$(P)DISABLE_IF_COMM_MODE")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:VALUE:OUTPUT:2")
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)VALUE:OUTPUT:BOTH")
+{
+    field(DESC, "Measurement value for both outputs")
+    field(DTYP, "stream")
+    field(PREC, "4")
+    field(EGU, "")
+    
+    field(INP, "@keylkg.proto getValueOutputBoth($(P)VALUE:OUTPUT:,1,2) $(PORT)")
```

Comment:
Removed. The device offered the ability to either capture one output at a time, or both. I thought it might be useful incase they wanted to capture both values at the same timestamp, rather than slightly apart; i.e. extended functionality over the VI.